src/payment-flow-handlers/donation-flow-modal-manager.ts

```diff
@@ -17,6 +17,17 @@ enum ModalHeaderColor {
   Red = '#691916',
 }
 
+/**
+ * The DonationFlowModalManager is responsible for most of the high-level modal flows.
```

Comment:
I really appreciate the distinction between flow manager and form controller.

Reply:
Thanks! 😄